mathgenerator/algebra.py

```diff
@@ -30,12 +31,19 @@ def calculate_gcd(x, y):
     elif a == 1 or a == i:
         x = f"{c - b}"
 
-    problem = f"${a}x + {b} = {c}$"
-    solution = f"${x}$"
+    #Have to add this and I don't know why
+    try:
+        if x*int(a) != int(c)-int(b):
+            x = f"{c - b}/{a}"
+    except:
+        return basic_algebra(max_variable)
```

Comment:
what is the purpose of this?

Reply:
The Reason is because there was a bug in the code and it would not correctly come up with a solution if x == 1.